tests/src/item.test.ts

```diff
@@ -0,0 +1,398 @@
+import * as anchor from "@project-serum/anchor";
+import { Transaction } from "@raindrop-studios/sol-kit";
+import { ItemProgram, Instructions } from "@raindrops-protocol/raindrops";
+import { IDL as ItemProgramIDL } from "./types/item";
+import { createMintNFTInstructions } from "./utils/token";
+
+describe("Item Program", () => {
+  it.concurrent("Item Flow", async () => {
+    const walletKeypair = anchor.web3.Keypair.generate();
+
+    const provider = new anchor.AnchorProvider(
+      new anchor.web3.Connection("http://localhost:8899"),
+      new anchor.Wallet(walletKeypair),
+      { commitment: "confirmed" }
+    );
+
+    await provider.connection.requestAirdrop(
+      walletKeypair.publicKey,
+      100 * anchor.web3.LAMPORTS_PER_SOL
+    );
+
+    const itemProgram = await ItemProgram.getProgramWithConfig(ItemProgram, {
+      asyncSigning: false,
+      provider,
+      idl: ItemProgramIDL,
+    });
+
+    let ixs, args, accounts, additionalArgs;
+
+    // STEP 1: Creates an item class
+
+    ixs = [];
+
+    const itemClassMintKeypair = anchor.web3.Keypair.generate();
+    const itemClassIndex = 0;
+
+    const createItemClassMintIxs = await createMintNFTInstructions(
+      provider,
+      itemClassMintKeypair.publicKey
+    );
+
+    const itemClassData = {
```

Comment:
Pull this outside of the test case. It's generic and bloats the test case's function body.

Reply:
IMO, we cannot put this outside as I mentioned below.

https://github.com/raindrops-protocol/raindrops/pull/34/files#r955302164

Reply:
There is no reason you can't pull this very large object outside of the test case function body and into a separate variable.